GitUpKit/Views/GIQuickViewController.h

```diff
@@ -16,7 +16,18 @@
 #import "GIViewController.h"
 
 @class GCHistoryCommit;
+@class GCCommit;
+@protocol GIQuickViewController__Delegate__Intentions
```

Comment:
Don't use underscores. End with "Delegate" if that's what it is.

Reply:
@lucasderraugh 

`GIQuickViewControllerIntentionsDelegate`
or
`GIQuickViewControllerDelegate`
?